src/modules/dashboard/components/NetworkContractUpgradeDialog/NetworkContractUpgradeDialogForm.tsx

```diff
@@ -0,0 +1,145 @@
+    defaultMessage: 'Current version',
+  },
+  newVersion: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.NetworkContractUpgradeDialog.NetworkContractUpgradeDialogForm.newVersion',
+    defaultMessage: 'New version',
+  },
+  annotation: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.NetworkContractUpgradeDialog.NetworkContractUpgradeDialogForm.annotation',
+    defaultMessage: 'Explain why you are upgrading this colony (optional)',
+  },
+  noPermissionText: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.NetworkContractUpgradeDialog.NetworkContractUpgradeDialogForm.noPermissionText',
+    defaultMessage: `You do not have the {requiredRole} permission required
+      to take this action.`,
+  },
+});
+
+interface Props {
+  back: () => void;
+  colony: Colony;
+}
+
+const NetworkContractUpgradeDialogForm = ({
+  back,
+  colony,
+  colony: { version },
+  handleSubmit,
+  isSubmitting,
+}: Props & FormikProps<FormValues>) => {
+  const { walletAddress, username, ethereal } = useLoggedInUser();
+
+  const allUserRoles = useTransformer(getAllUserRoles, [colony, walletAddress]);
+
+  const hasRegisteredProfile = !!username && !ethereal;
+  const canUpgradeVersion = hasRegisteredProfile && hasRoot(allUserRoles);
+
+  const { version: newVersion } = useNetworkContracts();
+
+  return (
+    <>
+      <DialogSection appearance={{ theme: 'heading' }}>
+        <Heading
+          appearance={{ size: 'medium', margin: 'none' }}
+          text={MSG.title}
+          className={styles.title}
+        />
+      </DialogSection>
+      {!canUpgradeVersion && (
+        <DialogSection>
+          <PermissionRequiredInfo requiredRoles={[ColonyRole.Root]} />
+        </DialogSection>
+      )}
+      <DialogSection>
+        <div className={styles.contractVersionLine}>
+          <FormattedMessage {...MSG.currentVersion} />
+          <div className={styles.contractVersionNumber}>{version}</div>
+        </div>
+        <div className={styles.contractVersionLine}>
+          <FormattedMessage {...MSG.newVersion} />
+          <div className={styles.contractVersionNumber}>
+            {newVersion === null ? version : newVersion}
+          </div>
+        </div>
+      </DialogSection>
+      <DialogSection>
+        <Annotations
+          label={MSG.annotation}
+          name="annotation"
+          disabled={!canUpgradeVersion}
+        />
+      </DialogSection>
+      {!canUpgradeVersion && (
+        <DialogSection>
+          <div className={styles.noPermissionMessage}>
+            <FormattedMessage
+              {...MSG.noPermissionText}
+              values={{
+                requiredRole: (
+                  <PermissionsLabel
+                    permission={ColonyRole.Root}
+                    name={{ id: `role.${ColonyRole.Root}` }}
+                  />
+                ),
+              }}
+            />
+          </div>
+        </DialogSection>
+      )}
```

Comment:
Please reduce the top padding of this dialog section so that the error has less empty space before it

![Screenshot from 2021-01-12 11-30-04](https://user-images.githubusercontent.com/1193222/104297359-73a1a600-54cb-11eb-9273-965b8e43abe7.png)

Reply:
The same padding is used in all other dialogs. Should we then reduce the top padding in all other dialogs as well for consistency purpose?